google/cloud/bigtable_v2/services/bigtable/transports/grpc_asyncio.py

```diff
@@ -512,6 +514,66 @@ def read_change_stream(
             )
         return self._stubs["read_change_stream"]
 
+    def _prep_wrapped_messages(self, client_info):
```

Comment:
how is is this called?

Reply:
It's overriding the [base class implementation](https://github.com/googleapis/python-bigtable/blob/94bfe664c5e9daa3bc76d6f423dd12b7949b2f76/google/cloud/bigtable_v2/services/bigtable/transports/base.py#L130) to use `gapic_v1.method_async.wrap_method` instead of `gapic_v1.method.wrap_method`